mandatory/1-fix-functions.js

```diff
@@ -21,7 +21,7 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num > 10;
 
   if (isBigEnough) {
     return "num is greater than 10";
```

Comment:
please change the variable "IsBigEnough"  to  "let IsBigEnough=num" and inside the if stament say IsBigEnough > 10 and the else stament to return num is less than 10

Reply:
I'm not following what seems to be the problem. I have  "let isBigEnough = num > 10;"  I have it this way so that anything that is less than 10 falls in the else statement and isBigEnough is where num > 10..

Reply:
is not a problem it's a suggestion
```suggestion
    if (isBigEnough > 10){
    return "num is greater than 10";
    }else {
    return "num is less than 10"
    }
```

Reply:
Oh!! I thought there were errors. Thank You for your suggestions. Much appreciated. Will try it out!